src/main/java/uk/gov/laa/ccms/caab/mapper/CopyApplicationMapper.java

```diff
@@ -24,25 +21,17 @@
 public interface CopyApplicationMapper {
 
   @BeanMapping(ignoreByDefault = true)
-  @Mapping(target = "caseReferenceNumber", source = "caseReferenceNumber")
-  @Mapping(target = "applicationType", source = "applicationToCopy.applicationType")
-  @Mapping(target = "providerDetails", source = "applicationToCopy.providerDetails")
-  @Mapping(target = "categoryOfLaw", source = "applicationToCopy.categoryOfLaw")
-  @Mapping(target = "correspondenceAddress", source = "applicationToCopy.correspondenceAddress")
-  @Mapping(target = "larScopeFlag", source = "applicationToCopy.larScopeFlag")
-  @Mapping(target = "proceedings", source = "applicationToCopy.proceedings")
-  @Mapping(target = "opponents", source = "applicationToCopy.opponents")
-  @Mapping(target = "client", source = "clientDetail")
-  @Mapping(target = "costs.requestedCostLimitation", source = "requestedCostLimitation")
-  @Mapping(target = "costs.defaultCostLimitation", source = "defaultCostLimitation")
-  @Mapping(target = "status", source = "status")
+  @Mapping(target = "applicationType", source = "applicationType")
+  @Mapping(target = "applicationType.devolvedPowers.contractFlag", ignore = true)
+  @Mapping(target = "providerDetails", source = "providerDetails")
+  @Mapping(target = "categoryOfLaw", source = "categoryOfLaw")
+  @Mapping(target = "correspondenceAddress", source = "correspondenceAddress")
+  @Mapping(target = "larScopeFlag", source = "larScopeFlag")
+  @Mapping(target = "proceedings", source = "proceedings")
+  @Mapping(target = "opponents", source = "opponents")
```

Comment:
Do these need to be mapped if they are the same value?

Reply:
Yes - I've double checked the mapstruct docs. 
`boolean ignoreByDefault
Default ignore all mappings. All mappings have to be defined manually. No automatic mapping will take place. No warning will be issued on missing source or target properties.`

That was the point of me using ignoreByDefault - so that I can control which bits of the applicationToCopy are mapped across.